packages/starlight/components/Head.astro

```diff
@@ -36,20 +36,18 @@ const headDefaults: z.input<ReturnType<typeof HeadConfigSchema>> = [
 			type: config.favicon.type,
 		},
 	},
+	// Twitter Tags
+	{
+		tag: 'meta',
+		attrs: { name: 'twitter:card', content: 'summary_large_image' },
+	},
```

Comment:
Was there a specific motivation for changing the order of this tag? I’d guess we can just leave it where it is to avoid introducing unnecessary churn?

Reply:
This is what X's (aka Twitter) documentation shows as an example, but I don't think it's strictly needed so if you want that tag to be there, I can accept this suggestion too. Thank you!